component/notification/src/main/java/org/exoplatform/social/notification/plugin/DlpAdminDetectedItemPlugin.java

```diff
@@ -84,38 +85,20 @@ public boolean isValid(NotificationContext ctx) {
         return true;
     }
 
-    private List<String> getRecipients() throws Exception {
-        HashSet<String> allGroupsMembers = new HashSet<>();
-
-        // Get admin members group
-        ListAccess<User> adminMembersAccess = organizationService.getUserHandler().findUsersByGroupId(userACL.getAdminGroups());
-        int totalAdminGroupMembersSize = adminMembersAccess.getSize();
-        User[] users = adminMembersAccess.load(0, totalAdminGroupMembersSize);
-        List<String> adminMembers = Arrays.stream(users)
-                                          .map(User::getUserName)
-                                          .collect(Collectors.toList());
-        allGroupsMembers.addAll(adminMembers);
-
-        // Get others members of groups
-        SettingValue<?> settingValue = settingService.get(Context.GLOBAL, Scope.APPLICATION.id("DlpPermissions"), "exo:dlpPermissions");
-        if(settingValue == null ||settingValue.getValue().toString().isEmpty()) return new ArrayList<String>(allGroupsMembers);
-        List<String> permissionsList = Arrays.asList(settingValue.getValue().toString().split(","));
-
-        for (String permission : permissionsList) {
-            if(permission.equals("/platform/administrators")) continue;
-            try {
-                ListAccess<User> membersAccess = organizationService.getUserHandler().findUsersByGroupId(permission);
-                int totalGroupMembersSize = adminMembersAccess.getSize();
-                User[] membersUsers = membersAccess.load(0, totalGroupMembersSize);
-                List<String> members = Arrays.stream(membersUsers)
-                                             .map(User::getUserName)
-                                             .collect(Collectors.toList());
-                allGroupsMembers.addAll(members);
-            } catch (Exception e){
-                LOGGER.error("Error when getting group");
-            }
+    private List<String> getRecipients() {
+        // Get DLP members group and super user
+        List<String> members = new ArrayList<>();
+        try {
+            ListAccess<User> adminMembersAccess = organizationService.getUserHandler().findUsersByGroupId(DLP_GROUP);
+            int totalAdminGroupMembersSize = adminMembersAccess.getSize();
+            User[] users = adminMembersAccess.load(0, totalAdminGroupMembersSize);
+            members = Arrays.stream(users)
+                            .map(User::getUserName)
+                            .collect(Collectors.toList());
+            members.add(userACL.getSuperUser());
```

Comment:
Why do we add super user ?

Reply:
superUser doit recevoir aussi notification puisqu'il a la DLP page par defaut.

Reply:
For me, the super user should not receive the notification if he is not in the dlp group.
Please ask to PO for clarification